test/modules/ModuleLib/ModuleUtil.cs

```diff
@@ -56,7 +56,7 @@ public static ILogger CreateLogger(string categoryName, LogEventLevel logEventLe
             return new LoggerFactory().AddSerilog().CreateLogger(categoryName);
         }
 
-        public static async Task ReportTestResultAsync(TestResultReportingClient apiClient, ILogger logger, TestResultBase testResult)
+        public static async Task ReportTestResultAsync(TestResultReportingClient apiClient, ILogger logger, TestResultBase testResult, bool isRethrow = false)
```

Comment:
Can we always call throw? and lets' caller to handle.

Reply:
It looks like there are multiple caller to this function that I don't want to make change right now. 
So, I'll make the default behavior a swallow exception. And if a new test module want the manage the exception, the caller can set the flag to be true.

Reply:
revert this fix.  I will take a look and better to handle in a separate PR.